docs/nimcache.md

```diff
@@ -24,19 +24,25 @@ Follow these steps to cache NIM models in a persistent volume.
 ```sh
 kubectl create ns nim-service
 ```
+### 2.Export NGC CLI API KEY
+`NOTE:` Ignore this step if you already export the NGC_CLI_API_KEY
 
-### 2. Create an Image Pull Secret for the NIM Container
+Please refer to get [NGC API Key](https://docs.nvidia.com/ngc/gpu-cloud/ngc-private-registry-user-guide/index.html#ngc-api-keys)
```

Comment:
nit: No "please" in technical documentation.

Reply:
@mikemckiernan done